src/components/NationalText/utils.ts

```diff
@@ -0,0 +1,62 @@
+import AggregationsJSON from 'assets/data/aggregations.json';
+import { last } from 'lodash';
+import moment from 'moment';
+import { formatPercent, formatDecimal } from 'common/utils';
+
+// Fix all uses of !
+
+const getPercentChange = (series: any[]): number => {
+  const lastTwoWeeks = series.slice(-14);
+  const difference = last(lastTwoWeeks)! - lastTwoWeeks[0]!;
+  const percentChange = difference / lastTwoWeeks[0]!;
+  return percentChange;
+};
+
+const getDescriptiorCopy = (percentChange: number): string => {
+  if (percentChange > 0) {
+    return 'increased ';
+  } else return 'decreased ';
+};
+
+const getTotalCasesCopy = (summedRawCases: number): string => {
+  const divided = summedRawCases / 1000000;
+  return `${formatDecimal(divided, 1)} million`;
+};
+
+export function getNationalText(): string {
```

Comment:
We might want to return a React element here in case we want to highlight the numbers, add links and the like. I don't think that's being required here, but it has happened a couple of times that the requirement comes in later :)